parcel-bundler/parcel
# Preventing an early exit when Ctrl+C is pressed more than once

Before:

Pressing Ctrl+C multiple times after a build caused Parcel to end while writing cache entries to disk. If you ran Parcel again, it would cause a v8 deserialization error.

Now:

Pressing Ctrl+C multiple times only ever enqueues a single exit.

Test Plan:

- Started a build
- Wait for 'Built in: ...' message 
- Press Ctrl+C multiple times
- Run the build again and verify that there is no deserialization error
